controls/radrichtextbox/overview.md

```diff
@@ -10,91 +10,98 @@ position: 0
+* __Track changes__
+	Support for tracking changes as formatting changes, adding and removing text, images, hyperlinks, tables, etc. This features is useful when different people are working on the same document. For more information, see the [Track Changes]({%slug radrichtextbox-features-track-changes%}) article.
 
-* {% if site.site_name == 'Silverlight' %}__Delete, Cut, Copy and Paste in plain text (using Silverlight 4.0 build-in clipboard). Copy/Paste in internal rich-text clipboard.__{% endif %}{% if site.site_name == 'WPF' %}__Rich-text clipboard –__ allows rich text copy/paste from Word, OpenOffice etc.; {% endif %}[Read more]({%slug radrichtextbox-features-clipboard-support%})
+* __Bookmarks__
+	The control provides easy retrieval of previously marked parts of the document. For more information, see the [Bookmarks]({%slug radrichtextbox-features-document-elements-bookmarks%}) article.
 
-* __Import/export__ - with Telerik __RadRichTextBox__ control you can load XAML, HTML, RTF, DocX (rich-text) or TXT (plain text) into the control, you can format and edit it and then export it back to any of the above formats or PDF. [Read more]({%slug radrichtextbox-features-import-export%})
+* __Comments__ 
+	RadRichTextBox offers the ability to add text to the document which is not inserted directly in the page. For more information, see the [Comments]({%slug radrichtextbox-features-comments%}) article.
 
-* __Headers and Footers__ - __RadRichTextBox__ supports headers and footers in its document. [Read more]({%slug radrichtextbox-features-headers-and-footers%})
+* __Document protection__
+	You can define editing restrictions in the document for a set of users and groups. For more information, see the [Document Protection]({%slug radrichtextbox-features-document-protection%}) article.
 
-* __Track changes__ - adding and removing of text, images, hyperlinks, tables, etc. and formatting changes of these elements can be tracked by different users. [Read more]({%slug radrichtextbox-features-track-changes%})
+* __Mail merge__ 
+	The control makes populating a template document with data easy. For more information, see the [Mail Merge]({%slug radrichtextbox-features-mail-merge%}) article.
 
-* __Bookmarks__ - provide easy retrieval of previously marked parts of the document. [Read more]({%slug radrichtextbox-features-document-elements-bookmarks%})
+* __Custom annotations__
+	You can extend the document model in order to serve specific needs. For more information, see the [Annotations Overview]({%slug radrichtextbox-features-document-elements-annotations%}) article.
 
-* __Comments__ - offer the ability to add text to the document which is not inserted directly in the page. [Read more]({%slug radrichtextbox-features-comments%})
+* __Footnotes and Endnotes__	

```

Comment:
I am not sure why but there is an additional blank line in the resultant article